packages/nextjs/app/builders/0xaC1bBf2Aba61D6fC6fCbade72eCe3362de4724c3/page.tsx

```diff
@@ -0,0 +1,107 @@
+// app/builder/address/page.tsx
+
+const AVATAR_URL = "https://avatars.githubusercontent.com/u/583231?v=4";
+const WALLET_ADDRESS = "0xaC1bBf2Aba61D6fC6fCbade72eCe3362de4724c3";
+
+const SOCIALS = [
+  { name: "Twitter", url: "https://twitter.com/yourhandle", icon: "🐦" },
+  { name: "GitHub", url: "https://github.com/yourhandle", icon: "💻" },
+  { name: "LinkedIn", url: "https://linkedin.com/in/yourhandle", icon: "🔗" },
+];
+
+export default function Web3ProfilePage() {
```

Comment:
You might want to consider typing your page as NextPage since we are using TypeScript, as this ensures better integration and adds clarity for future maintainers by indicating that the component is a Next.js page.